operations/helm/scripts/build.sh

```diff
@@ -64,6 +64,11 @@ function generate_manifests() {
   INTERMEDIATE_OUTPUT_DIR=$2
   OUTPUT_DIR=$3
 
+  # Get the pid of the subshell calling this function in a portable way.
+  # Do not use BASHPID come some shells don't support it.
+  # shellcheck disable=SC2016
+  PID=$(exec sh -c 'echo "$PPID"')
```

Comment:
Note: the shell script linter complains that `$PPID` is not evaluated in the script, which is exactly what we want because we want to evaluate it by the `exec sh`. For this reason I've added a rule to disable the check.